webapps-schema/src/main/java/io/camunda/webapps/schema/entities/tasklist/MetricEntity.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.schema.entities.tasklist;
+
+import java.time.OffsetDateTime;
+import java.util.Objects;
+
+public class MetricEntity extends TasklistEntity<MetricEntity> {
```

Comment:
I just added a similar one for operate in https://github.com/camunda/camunda/pull/23690, do we need two of the same?

Reply:
So what we need is having two indices. Entities we might be able to streamline?

Reply:
in general can we maybe also just use one entity base class?

Reply:
Yeah probably we can use the same entity, since the index name is being passed to the handler directly. I will reiterate on this one, once @mustafadagher  's PR is merged.